cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/replication/ReplicationPolicyManager.java

```diff
@@ -28,6 +28,9 @@
         void replicationErrored(int id);
     }
 
+    /**
```

Comment:
After talking to @brynh I think we should probably add a comment here pointing people at the `ReplicationsCompletedListener` so that if they think this is the class they want they can find what they really want :)